landing-pages/site/content/en/blog/airflow-2.10.0/index.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "Apache Airflow 2.10.0 is here"
+linkTitle: "Apache Airflow 2.10.0 is here"
+author: "Utkarsh Sharma"
+github: "utkarsharma2"
+linkedin: "utkarsh-sharma-5791ab8a"
+description: "Apache Airflow 2.10.0 is a game-changer, with powerful Dataset improvements and the groundbreaking Hybrid Executor, set to redefine your workflow capabilities!"
+tags: [Release]
+date: "2024-08-08"
+---
+
+I'm happy to announce that Apache Airflow 2.10.0 is now available, bringing an array of noteworthy enhancements and new features that will greatly serve our community.
+
+Apache Airflow 2.10.0 contains over 135 commits, which include 43 new features,  85 improvements, 43 bug fixes, and 26 documentation changes.
```

Comment:
The numbers seem off. I need to figure out how to get the proper data.

Reply:
We normally just get the counts from the release notes. Close enough.

Reply:
@jedcunningham I updated the count using [git compare](https://github.com/apache/airflow/compare/2.9.0...2.10.0rc1). PTAL

Reply:
My gut tells me you've swung too far in the other direction. Trying to get just core commits, roughly.